kratos/utilities/atomic_utilities.h

```diff
@@ -0,0 +1,64 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//                   Denis Demidov
+//
+
+#if !defined(KRATOS_ATOMIC_UTILITIES_H_INCLUDED )
+#define  KRATOS_ATOMIC_UTILITIES_H_INCLUDED
+
+
+// System includes
+#include<omp.h>
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+
+namespace Kratos
+{
+///@addtogroup KratosCore
```

Comment:
can you please add a brief description what this is / how it should be used?

Reply:
done